src/build/genprod.scala

```diff
@@ -172,19 +175,13 @@ object Function {
 class Function(val i: Int) extends Group("Function") with Arity {
   def descriptiveComment  = ""
   def functionNTemplate =
-"""
+s"""
```

Comment:
Thanks for catching that, that's not necessary anymore as the comment is only used for Function1. Removed it!